<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog</class>
 <widget class="QDialog" name="Dialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1200</width>
    <height>800</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>480</x>
     <y>240</y>
     <width>281</width>
     <height>51</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Sign in to your existing account</string>
   </property>
  </widget>
  <widget class="QWidget" name="widget" native="true">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>-10</y>
     <width>1211</width>
     <height>811</height>
    </rect>
   </property>
   <property name="mouseTracking">
    <bool>true</bool>
   </property>
   <property name="focusPolicy">
    <enum>Qt::NoFocus</enum>
   </property>
   <property name="acceptDrops">
    <bool>false</bool>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(255, 255, 255);</string>
   </property>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>550</x>
      <y>180</y>
      <width>161</width>
      <height>81</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>36</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Login</string>
    </property>
   </widget>
   <widget class="QPushButton" name="Log_In">
    <property name="geometry">
     <rect>
      <x>470</x>
      <y>530</y>
      <width>311</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>14</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">border-radius: 20px;
background-color: rgb(0, 0, 0);
color:white</string>
    </property>
    <property name="text">
     <string>Log in</string>
    </property>
    <property name="shortcut">
     <string>Return</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="emailfield">
    <property name="geometry">
     <rect>
      <x>480</x>
      <y>350</y>
      <width>281</width>
      <height>41</height>
     </rect>
    </property>
    <property name="toolTip">
     <string/>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgba(0, 0, 0,0);</string>
    </property>
    <property name="placeholderText">
     <string>Username</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="passwordfield">
    <property name="geometry">
     <rect>
      <x>480</x>
      <y>440</y>
      <width>281</width>
      <height>41</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgba(0, 0, 0,0);</string>
    </property>
    <property name="placeholderText">
     <string>Password</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>480</x>
      <y>330</y>
      <width>81</width>
      <height>16</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>10</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Username</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_4">
    <property name="geometry">
     <rect>
      <x>480</x>
      <y>420</y>
      <width>81</width>
      <height>16</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>10</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Password</string>
    </property>
   </widget>
   <widget class="QLabel" name="error">
    <property name="geometry">
     <rect>
      <x>480</x>
      <y>490</y>
      <width>281</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">color:red;</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="back">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>730</y>
      <width>51</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>14</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">border-radius: 20px;
background-color: rgba(0, 0, 0,0);
color:white</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="imageholder">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>730</y>
      <width>51</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <zorder>imageholder</zorder>
   <zorder>label</zorder>
   <zorder>Log_In</zorder>
   <zorder>emailfield</zorder>
   <zorder>passwordfield</zorder>
   <zorder>label_3</zorder>
   <zorder>label_4</zorder>
   <zorder>error</zorder>
   <zorder>back</zorder>
  </widget>
  <zorder>widget</zorder>
  <zorder>label_2</zorder>
 </widget>
 <resources/>
 <connections/>
</ui>
